packages/ai-native/src/browser/types.ts

```
@@ -360,6 +360,53 @@
+  appendLine(message: string): void;
+}
+
+export interface MCPToolDefinition {
+  name: string;
+  description: string;
+  inputSchema: any; // JSON Schema
+  handler: (
+    args: any,
+    logger: MCPLogger,
+  ) => Promise<{
+    content: { type: string; text: string }[];
```

Biome (1.9.4)

[error] Shouldn't redeclare 'MCPServerContribution'. Consider to delete it or rename it.

'MCPServerContribution' is defined here:


(lint/suspicious/noRedeclare)